### ISSUE TITLE
Implement basic user admin page

### ISSUE THREAD

**Issue by diox** [mozilla/addons-server#8191]
Split from #7268, this issue is about implementing the base user admin page without the special actions that will be available depending on permissions (those will be implemented in a separate issue after this one is done).

Users with `Users:Edit` should see all user data in the admin page:
  * User ID.
  * Email address.
  * Link to profile page.
  * Metadata: username, description, image, location.
  * Last time of activity.
  * Ratings posted.
  * Collections, public or private.
  * Add-ons created, regardless of status and type.
  * Activity history.
  * Known IP addresses (last login, and used for ratings).
  * Abuse reports reported *by* this user.
  * Abuse reports reported *for* this user.

**Comment by jvillalobos** [comment]
The following should be editable: username, display name, description, location, image (just deleting the existing image is enough).

**Comment by diox** [comment]
Note : image deletion will be implemented with other actions in #7268, this is only the basic foundation.

**Comment by AlexandraMoga** [comment]
I have verified this on AMO-dev with an Admin user and a user from Staff (group having `Users:Edit` permission):
- all the functionalities mentioned in the above list are present
- requested editable fields (except for picture deletion which is not implemented here) were implemented
- changes applied to the user profile through the Admin pages are successfully applied

Questions for @diox :
1. The link to the AMO user profile page is currently redirecting to the legacy site (see https://addons-internal.dev.mozaws.net/en-US/firefox/user/basket-dev-new/) Should this also be changed to point to the new frontend once the User profile pages are fully functional?
2. In the list of required editable fields mentioned above, the E-mail address was not included. However, this can be edited from the user Admin page. Is this expected?
3. When accessing the page with a user from `Staff`, I don't have access to the list with user's addons, collections, ratings etc (ex. https://addons-internal.dev.mozaws.net/en-US/admin/models/addons/addon/?authors=10641644). Are these the 'special actions` to be implemented separately  you were referring to in the opening of this issue?
